back/spec/interactors/confirm_user_spec.rb

```diff
@@ -65,7 +65,7 @@
     end
 
     it 'returns a code invalid error' do
-      expect(result.errors.details).to eq(code: [{ error: :expired }])
+      expect(result.errors.details).to eq(code: [{ error: :invalid }])
```

Comment:
As you can see from the description, the expected code used to be `invalid`. This was changed by another developer while he was upgrading Rails, so there doesn't seem to have been a good reason for this change. I therefore suspect that the `invalid` code is more desired in this case. We have the `too_many_retries` code in place to avoid brute-force guessing of the confirmation code.

Reply:
I think I changed the order but I need to go back and understand why.

Reply:
According to git blame, it was another developer (Sergey)

Reply:
Aha no, what I changed was the order of code value and retry count. https://github.com/CitizenLabDotCo/citizenlab/pull/4072/files#diff-3bc35be76fb17fa29424ada65881547fb8539bf04935b2ddaae39b51f66489e1